packages/auth-compat/test/helpers/helpers.ts

```diff
@@ -17,6 +17,7 @@
 
 import * as sinon from 'sinon';
 import firebase from '@firebase/app-compat';
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This should have a corresponding eslint-enabled after the "errant" statement below.